### PR TITLE
Reduce the memory footprint of 'ecma_instantiate_builtin'

### PR DESCRIPTION
**Pull request by LaszloLango**
JerryScript-DCO-1.0-Signed-off-by: László Langó llango.u-szeged@partner.samsung.com

### CODE REVIEW

**Comment by LaszloLango** [comment]
Binary sizes (bytes)
fbf5c32747:134368
b8d427e558:133860

File: jerry-core/ecma/builtin-objects/ecma-builtins.c [hunk 1]
```diff
@@ -289,6 +289,40 @@ ecma_builtin_init_object (ecma_builtin_id_t obj_builtin_id, /**< built-in ID */
   return obj_p;
 } /* ecma_builtin_init_object */
 
+
+/**
+ * Helper function for 'ecma_instantiate_builtin'
+ */
+static void ecma_instantiate_builtin_helper (ecma_builtin_id_t builtin_id, /**< built-in id */
```
Comment (zherczeg) [inline jerry-core/ecma/builtin-objects/ecma-builtins.c]: put a newline after `static void`

Comment (LaszloLango) [reply jerry-core/ecma/builtin-objects/ecma-builtins.c]: fixed

**Comment by robertsipka** [comment]
LGTM (informally)

**Pull request merged**